.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,37 +1,28 @@
 # aws-doc-sdk-examples Pull Request
 
-Thank you for making a submission to the *aws-doc-sdk-examples* repository. For more information about submitting pull requests to this repository, see [Guidelines for contributing](/CONTRIBUTING.md).
+Thank you for making a submission to the *aws-doc-sdk-examples* repository. For more information about submitting pull requests to this repository, see [Guidelines for contributing](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/CONTRIBUTING.md).
 
-*NOTE:* This PR template contains three sections. Depending on the reason for your pull request, please fill out the section that applies to you and then remove the other two sections.
+*NOTE:* This pull request (PR) template contains three sections. Please remove the two sections that don't apply to your submission.
 
-## New SDK Code Example
+## I am submitting a new SDK code example.
 
-The submitter has:
+Please confirm you have met the following minimum requirements::
 
 - [ ] Added the default copyright notice to all files.
-- [ ] Created unit tests for all paths through the code and they all pass.
+- [ ] Created unit tests for all paths through the code, and confirmed that the all the tests pass. See [How we test code examples](https://github.com/awsdocs/aws-doc-sdk-examples/wiki/Code-quality-guidelines---testing-and-linting#how-we-test-code-examples) for recommendations.
 - [ ] Run a linter against all code and implemented the resulting suggestions.
 - [ ] Added minimum usage documentation as comments in the code.
-- [ ] Had comments and strings reviewed and incorporated suggested changes.
-- [ ] Had the code reviewed and implemented the reviewer's suggested changes.
 
-## Existing Example Update
+## I am resolving an issue with an existing code example.
 
-The submitter has:
+Please describe the changes you have made here, including any issue numbers.
 
-- [ ] Confirmed that the correct copyright is included in all files.
-- [ ] Major code changes have been reviewed, and the submitter has incorporated review comments.
-- [ ] Changed or added comments and strings have been reviewed, and the submitter has incorporated any and all suggested edits.
+Please confirm you have met the following minimum requirements::
 
-## Resolve Issue
+- [ ] Tested the changed code. See [How we test code examples](https://github.com/awsdocs/aws-doc-sdk-examples/wiki/Code-quality-guidelines---testing-and-linting#how-we-test-code-examples) for recommendations.
```

Comment:
- [ ] Test the changed code. For recommendations, see [How we test code examples](https://github.com/awsdocs/aws-doc-sdk-examples/wiki/Code-quality-guidelines---testing-and-linting#how-we-test-code-examples).